docs/content/configuration/creating_volumestorageclass.md

```diff
@@ -2,36 +2,45 @@
 |IBM FlashSystem® A9000 and A9000R|Always includes deduplication and compression. No need to specify during configuration.|
-|IBM Spectrum® Virtualize Family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `deduplicated` <br /><br /> **Note:** If not specified, the default value is thick.|
-|IBM® DS8000® Family| - `none` (default value) <br />- `thin`<br /><br /> **Note:** If not specified, the default value is `none`.|
+|IBM Spectrum® Virtualize family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `dedup_thin` (creates volumes that are deduplicated with thin-provisioning)<br />- `dedup_compressed` (creates deduplicated and compressed volumes)<br /><br /> **Note:** <br />- The `deduplicated` value is deprecated. Use `dedup_compressed`, if possible. When used, `deduplicated` provide the same results as `dedup_compressed`.<br />- If not specified, the default value is `thick`.|
+|IBM® DS8000® family| - `none` (default value) <br />- `thin`<br /><br /> **Note:** If not specified, the default value is `none`.|
 
-- The IBM DS8000 Family `pool` value is the pool ID and not the pool name as is used in other storage systems.
+- The IBM DS8000 family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
+- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
+  
+  ```
+  pool: demo-pool1:demo-pool2 
+  ```
+   **Important:** The two pools must be from different sites.
+   
+  For more information about the HyperSwap limitations and requirements, see [Limitations](../release_notes/limitations.md) within the release notes and [Compatibility and requirements](../installation/install_compatibility_requirements.md) within this guide.
+
 - The `allowVolumeExpansion` parameter is optional but is necessary for using volume expansion. The default value is _false_.
 
-**Note:** Be sure to set the value to true to allow volume expansion.
+  **Note:** Be sure to set the value to true to allow volume expansion.
 
 - The `csi.storage.k8s.io/fstype` parameter is optional. The values that are allowed are _ext4_ or _xfs_. The default value is _ext4_.
 - The `volume_name_prefix` parameter is optional.
 
-**Note:** For IBM DS8000 Family, the maximum prefix length is 5 characters. The maximum prefix length for other systems is 20 characters. <br /><br />For storage systems that use Spectrum Virtualize, the `CSI` prefix is added as default if not specified by the user.
+  **Note:** For IBM DS8000 family, the maximum prefix length is five characters. The maximum prefix length for other systems is 20 characters. <br /><br />For storage systems that use Spectrum Virtualize, the `CSI` prefix is added as default if not specified by the user.
 
     
     kind: StorageClass
```

Comment:
I think this needs to be indented as well in order to keep the "code block" formatting

Reply:
fixed section formatting